features/pipeline-token.feature

```diff
@@ -0,0 +1,75 @@
+@pipelinetoken
+Feature: Pipeline Token
+
+  A Pipeline token can be created and used in place of a password when
+    interacting with the regarding pipeline API. This would allow users to obtain valid JWTs and
+    interact with the Screwdriver API in a programmatic way.
+
+  Rules:
+      - Tokens should be specifically tied to the user that issued them
+      - Tokens should have a label/name associated with them
+      - Users can add description to their issued, so they know what they're using them for
+      - User can see when a token was last used
+      - Users can revoke a token at any time
+      - No one can ever look up the raw value after initially generating the token
+
+  Background:
+    Given: "calvin" is logged in
+  @ignore
+  Scenario: Generate New Pipeline Token
+    Given "calvin" dose not own a token named "tiger"
```

Comment:
I think pipeline token is owned by a pipleine, not a user.
"calivin" onws a pipeline, and the pipeline owns pipeline token.

Reply:
I think so.
I should fix wording.